week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,7 +5,14 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(strArr) {
+  for( let i=0; i< strArr.length; i++){
+    strArr[i]= strArr[i].trim().replace('/','').toLowerCase()
```

Comment:
What if there is more than 1 `/` e.g. `/path/to/files`?

Reply:
strArr[i].trim().replace(/\//g,'').toLowerCase()

Reply:
I've just tried to run that, I get an error
https://repl.it/repls/DeepIllustriousBrackets